docker/README.md

```diff
@@ -42,6 +42,8 @@ Port information:
 
 * __80__: Default controller web UI port.
 
+* __16001__: Controller port for agent connection.
```

Comment:
thx, it is set automatically in agent configurations when you download the agent.